spec/mock-request.js

```diff
@@ -1,14 +1,26 @@
 "use strict";
 const q = require("q");
+let clock = jasmine.Clock;
```

Comment:
I was following the way it was done in `realtime-callbacks.spec.js`, /agree referencing it explicitly is better.